docs/source/polars-cloud/organization/billing.md

```diff
@@ -1,27 +1,29 @@
-# Payment and billing via AWS Marketplace
+# Payment and billing
 
 All payments and billing for Polars Cloud are handled via the AWS Marketplace. This integration
 provides a secure, streamlined way to manage your subscription, payments, and billing using AWS's
 trusted marketplace infrastructure.
 
-## Subscribe to Polars Cloud
+!!! tip "30-day Free Trial"
+
+    New organizatoins get full access to explore Polars Cloud for 30 days. The trial activates automatically when connecting your first workspace to AWS. After the trial you can easily subscribe via the AWS Marketplace.
 
-### Prerequisites
+## Pricing
 
-Before subscribing to Polars Cloud through AWS Marketplace, ensure you have:
+Polars Cloud uses usage-based pricing charged per vCPU hour of compute usage. Billing begins when
+instances are booted and ready to execute queries, and ends when users stop the instances or when we
+detect instances have been terminated externally. Instance startup time and shutdown time after
+stopping are not charged.
 
-- An active AWS account with billing permissions
+## Subscribe to Polars Cloud
 
-### Steps to Sign Up
+Subscribing to Polars Cloud requires an AWS account with billing permissions. You can access the
+marketplace listing either through the Billing page in your Polars Cloud organization or by
+searching "Polars Cloud" directly in AWS Marketplace.
 
-1. Access the Marketplace: If you already have a Polars Cloud account, select the link to AWS
-   Marketplace on the `Billing` page of your organization. Alternatively, go directly to AWS
-   Marketplace and search for `Polars Cloud`.
-2. Review the Listing: Select the Polars Cloud listing and then select `View purchase options`.
-3. Review Terms: Carefully review the pricing, offer details, and terms and conditions.
-4. Subscribe: Select `Subscribe` at the end of the page to complete your subscription.
-5. Connect Your Account: Finalize the connection by selecting `Set up your account` in the
-   Marketplace portal.
+The subscription process connects your Polars Cloud organization to AWS billing, after which your
+organization status updates to reflect the marketplace connection. This typically completes within
+minutes, though allow up to an hour for status propagation.
```

Comment:
I think we should specify maybe 15 minutes. I've never seen it take more than 1-2 minutes so after 15 something is definitely wrong